Lykke.Service.LykkeService/src/Lykke.Service.LykkeService/Controllers/IsAliveController.cs

```diff
@@ -12,23 +14,18 @@ public class IsAliveController : Controller
         /// <summary>
         /// Checks service is alive
         /// </summary>
-        /// <returns></returns>
         [HttpGet]
         [SwaggerOperation("IsAlive")]
-        [ProducesResponseType(typeof(IsAliveResponse), (int)HttpStatusCode.OK)]        
+        [ProducesResponseType(typeof(IsAliveResponse), (int)HttpStatusCode.OK)]
         public IActionResult Get()
         {
             // NOTE: Feel free to extend IsAliveResponse, to display job-specific indicators
             return Ok(new IsAliveResponse
             {
-                Name = Microsoft.Extensions.PlatformAbstractions.PlatformServices.Default.Application.ApplicationName,
-                Version = Microsoft.Extensions.PlatformAbstractions.PlatformServices.Default.Application.ApplicationVersion,
-                Env = Program.EnvInfo,
-//#$if DEBUG
-                IsDebug = true,
-//#$else
-                //$#$//IsDebug = false,
-//#$endif
+                Name = AppEnvironment.Name,
+                Version = AppEnvironment.Version,
+                Env = AppEnvironment.EnvInfo,
+                IsDebug = Debugger.IsAttached,
```

Comment:
Afaik, this indicates if debugger is attached to the process, but not DEBUG/RELEASE build configuration

Reply:
Agree - let's use implementation from https://github.com/LykkeCity/Lykke.Sdk/pull/27